### PR TITLE
FRW-7373 Non-transactional mode for Data Exchange API.

### PR DESCRIPTION
**Pull request by yaroslav-spryker**
## PR Description

https://spryker.atlassian.net/browse/FRW-7373

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md [hunk 1]
```diff
@@ -602,7 +602,60 @@ to the child entities based on the newly created parent entity's ID.
 
 {% endinfo_block %}
 
-#### Error codes
```
Comment (asmarovydlo) [inline docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md:605]: https://github.com/spryker/suite-nonsplit/pull/8817/files glossary changes should be added to integration guides

File: docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md [hunk 2]
```diff
@@ -602,7 +602,60 @@ to the child entities based on the newly created parent entity's ID.
...
+## Non-transactional saving
+
+By default, the Data Exchange API uses a transactional approach to save data.
```
Comment (asmarovydlo) [inline docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md:607]: Please add response example somewhere

File: docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md [hunk 3]
```diff
@@ -602,7 +602,60 @@ to the child entities based on the newly created parent entity's ID.
...
+This means that if an error occurs during the saving process, the entire transaction is rolled back, and no data is saved. 
+However, in some cases, you may want to save data non-transactionally. In no-transactional mode, the API wraps each entity and its related records (if present in the request) in a separate transaction.
```
Comment (asmarovydlo) [inline docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md:609]: Please mention how exactly the data will be saved, and what will not be saved in this case.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md [hunk 4]
```diff
@@ -602,23 +602,124 @@ to the child entities based on the newly created parent entity's ID.
 
 {% endinfo_block %}
 
-#### Error codes
```
Comment (asmarovydlo) [inline docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md:605]: https://github.com/spryker/spryker-docs/pull/2690/files#r1625648326 not fixed

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md (hunk 1 above)
Comment (yaroslav-spryker) [reply docs/pbc/all/data-exchange/202404.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md:605]: IG was updated as part of another PR for DELETE https://github.com/spryker/spryker-docs/pull/2686

**Review by asmarovydlo — DISMISSED** [review]

**Pull request merged**